modules/post/windows/gather/credentials/filezilla_server.rb

```diff
@@ -236,7 +241,7 @@ def get_filezilla_creds(paths)
       ]
     end
 
-    session.db_record ? (source_id = session.db_record.id) : (source_id = nil)
+    session.db_record ? session.db_record.id : nil
```

Comment:
This looks like dead code

(Unless it's got some sort of weird side effect that isn't obvious)

Reply:
Removed